### PR TITLE
Fetch actions on document status change

### PR DESCRIPTION
**Pull request by siemiatj**
Related to #2469

### CODE REVIEW

**Comment by metas-ts** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/metasfresh-webui-frontend/metasfresh-webui-frontend/pullRequest?prid=4601260&bid=15442444)

File: src/components/widget/ActionButton.js [hunk 1]
```diff
@@ -127,12 +128,16 @@ class ActionButton extends Component {
    * @todo Write the documentation
    */
   handleChangeStatus = status => {
-    const { onChange } = this.props;
+    const { onChange, docId, windowType, activeTab } = this.props;
```
Comment (metas-ts) [inline src/components/widget/ActionButton.js]: Codacy Issue found: ['activeTab' is missing in props validation (react/prop-types)](https://app.codacy.com/app/metasfresh/metasfresh-webui-frontend/pullRequest?prid=4601260)

**Comment by siemiatj** [comment]
No idea what was I thinking. I did fetched the updated actions, buttons disappeared but not the one I was after all along. nvm, this just needs to be rewritten like quick actions and use redux store as data source.

**Comment by metas-ts** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/reducers/windowHandler.js  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/metasfresh/metasfresh-webui-frontend/pullRequest?prid=4601260&bid=15442444)

**Pull request merged**